kafka/producer/vendor/github.com/microsoft/confidential-container-demos/kafka/util/util.go

```diff
@@ -0,0 +1,15 @@
+package util
```

Comment:
I'm not so sure we're expecting this vendor file to exist after you add the replace declarations.  After you update the mod files, delete this directory from the vendor list and double check.